internal/provisioner/plain/controllers/bundle_controller.go

```diff
@@ -266,16 +265,29 @@ func (r *BundleReconciler) handleCompletedPod(ctx context.Context, u *updater.Up
 		return updateStatusUnpackFailing(u, fmt.Errorf("get bundle image digest: %w", err))
 	}
 
-	objects, err := getObjects(bundleFS)
+	bundleMetadata, err := bundle.MetadataFromFS(bundleFS)
 	if err != nil {
-		return updateStatusUnpackFailing(u, fmt.Errorf("get objects from bundle manifests: %w", err))
+		if !errors.Is(err, bundle.ErrBundleFileNotFound) {
+			return updateStatusUnpackFailing(u, fmt.Errorf("read bundle metadata from filesystem: %w", err))
+		}
+		// If we didn't find the bundle.yaml file, assume it's a plain+v0 bundle
```

Comment:
Might be helpful to emit a log here stating that an assumption was made regarding the bundle schema?